### PR TITLE
Added snackbar css

### PR DESCRIPTION
**Pull request by tanishq20**
Added different variants of snackbar

- Leading snackbar
- Baseline snackbar
- Stacked snackbar

Live preview link: https://deploy-preview-15--compact-ui.netlify.app/docs/components/snackbar

### CODE REVIEW

File: docs/components/snackbar.html [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+            </main>
+        </div>
+    </div>
+
+    <div class="hidden" id="leadingClass">
+        <div class="snackbar-container flex left rounded-5">
+            <p class="snackbar-content">Your photo has been archived.</p>
+            <div class="snackbar-action">
+                <a href="#" class="snackbar-action-item">RETRY</a>
+                <a href="" class="snackbar-action-item" id="leadingClose"><i class="fas fa-times"></i></a>
```
Comment (rahulcs754) [inline docs/components/snackbar.html]: set # in href

File: docs/components/snackbar.html [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+            </div>
+        </div>
+    </div>
+
+    <div class="flex justify-center hidden" id="baslineClass">
+        <div class="snackbar-container flex rounded-5">
+            <p class="snackbar-content">Can't send photo. Retry in 5 seconds.</p>
+            <div class="snackbar-action">
+                <a href="#" class="snackbar-action-item">RETRY</a>
+                <a href="" class="snackbar-action-item" id="baslineClose"><i class="fas fa-times"></i></a>
```
Comment (rahulcs754) [inline docs/components/snackbar.html]: same as above

File: docs/components/snackbar.html [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+            </div>
+        </div>
+    </div>
+
+    <div class="hidden" id="stackedClass">
+        <div class="snackbar-container right rounded-5">
+            <p class="snackbar-content">Can't send photo. Retry in 5 seconds.</p>
+            <div class="snackbar-action">
+                <a href="#" class="snackbar-action-item">RETRY</a>
+                <a href="" class="snackbar-action-item" id="stackedClose"><i class="fas fa-times"></i></a>
```
Comment (rahulcs754) [inline docs/components/snackbar.html]: same as above

File: docs/js/snackbar.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+// Close snackbar EventListener
+leadingClose.addEventListener('click', closeLeading)
+baslineClose.addEventListener('click', closeBasline)
+stackedClose.addEventListener('click', closeStacked)
+
+// Show clicked snackbar
+function showLeading() {
+  leadingClass.classList.remove('hidden')
+  setTimeout(() => {
```
Comment (rahulcs754) [inline docs/js/snackbar.js]: you can write this code like this

 setTimeout(() => leadingClass.classList.add('hidden') , 3000)

File: docs/js/snackbar.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+}
+function showBasline() {
+  baslineClass.classList.remove('hidden')
+  setTimeout(() => {
+    baslineClass.classList.add('hidden')
+  }, 3000)
+}
+function showStacked() {
+  stackedClass.classList.remove('hidden')
+  setTimeout(() => {
```
Comment (rahulcs754) [inline docs/js/snackbar.js]: same as above

**Review by rahulcs754 — COMMENTED** [review]
all look good 
just some change

File: stylesheets/components/snackbar.css [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  font-size: 2rem;
+  color: var(--black-500);
+  padding: 0.5rem;
+  height: 3rem;
+  width: 3rem;
+  text-align: center;
+}
+
+.snackbar-action-item i:hover {
+  background-color: rgb(0, 0, 0, 0.1);
```
Comment (shuntzhere) [inline stylesheets/components/snackbar.css]: You can use some variables here to do the same.

**Review by shuntzhere — APPROVED** [review]
Suggested a minor update.

**Review by rahulcs754 — APPROVED** [review]

**Pull request merged**